File: src/test/java/de/lukaseichler/recurrentneuralnetwork/NeuralNetworkTest.java
package de.lukaseichler.recurrentneuralnetwork;

import de.lukaseichler.recurrentneuralnetwork.activation.LogActivation;

/**
 * @author leichler
 */
public class NeuralNetworkTest {

    protected double applyDefaultActivationFunction(final double value) {
        return new LogActivation().apply(value);
    }

}

File: src/test/scala/de/lukaseichler/recurrentneuralnetwork/NeuralNetworkTest.scala
package de.lukaseichler.recurrentneuralnetwork

import de.lukaseichler.recurrentneuralnetwork.activation.LogActivation

/**
 * @author Lukas Eichler
 */
trait NeuralNetworkTest {
  protected def applyDefaultActivationFunction(value: Double): Double = {
    new LogActivation().apply(value)
  }
}
